### DISCUSSION TITLE
Spatial temperature profiles

### DISCUSSION THREAD

**Discussion by cooluser26** [epochpic/epoch#757] [Q&A] [answered]
Hello again,

Is it possible to set a spatial temperature profile for a species? Would it work with something like temp = Te0 * exp(-x/x0) ? 

Thanks a lot,

William

**Comment by Status-Mirror** [comment, marked as answer]
Hey William,

The temperature block does allow for spatial distributions. You can check out our [momentum loading demo](https://epochpic.github.io/quickstart/basic_examples/particle_load.html#momentum-distributions) to see how the temperature is converted to a momentum distribution of particles in a given cell.

While our examples use uniform temperature profiles, spatially-varying profiles are also allowed.

Hope this helps,
Stuart
